src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableConstants.cs

```diff
@@ -22,6 +22,9 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableD
 {
     internal static class HotspotsTableConstants
     {
+        public static readonly string TableDisplayName = Resources.HotspotsToolWindowCaption;
         public const string TableManagerIdentifier = nameof(TableManagerIdentifier);
+        public const string TableIdentifier = nameof(TableIdentifier);
```

Comment:
Am I missing something - the identifier for the table is the string "TableIdentifier"? Same for the type identifier.

Those aren't very descriptive names and there's nothing to suggest they relate to SonarLint. I'm guessing those identifiers are ones we're providing to VS so consumers can get hold of the table. If so, then we also want to reduce the possibility of name conflicts.
Either way, I think the identifiers should have some reference to SonarLint.